cypress/support/pages/discounts/vouchersPage.js

```diff
@@ -27,3 +28,29 @@ export function createVoucher({
     .click()
     .confirmationMessageShouldDisappear();
 }
+
+export function setVoucherDate({
+  voucherId,
+  startDate,
+  endDate,
+  endTime,
+  hasEndDate = false
+}) {
+  cy.visit(voucherDetailsUrl(voucherId)).waitForProgressBarToNotBeVisible();
+  if (startDate) {
+    cy.get(VOUCHERS_SELECTORS.startDate).type(startDate);
+  }
+  if (endDate) {
+    if (hasEndDate) {
+      cy.get(VOUCHERS_SELECTORS.hasEndDateCheckbox).click();
+    }
+    cy.get(VOUCHERS_SELECTORS.endDateInput)
+      .type(endDate)
+      .get(VOUCHERS_SELECTORS.endTimeInput)
+      .type(endTime);
```

Comment:
If it is under 10 minutes then an error is shown because of wrong format.
![image](https://user-images.githubusercontent.com/29308910/151950858-2d9c210d-a806-41f0-a3ce-4c750b056b3f.png)